### DISCUSSION TITLE
[Enhancement] [select] 希望能在Select组件支持一个Input插槽

### DISCUSSION THREAD

**Discussion by lhz239996423** [element-plus/element-plus#19172] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

## Description

在某些场景下，Select的显示不是一定与列表关联的，列表只是展示与匹配搜索值，此时如果想要更改SELECT的显示值，甚至是保留上一次的历史录入的场景，在获取焦点后能接着历史值编辑，但以上目前都无法做到。
希望能像viewUI类似的Select组件一样，支持一个input插槽，可以给开发人员自定义组件录入功能，目前刚升级ElementUI，非常需要该支持

### Additional comments

_(empty)_

**Comment by lhz239996423** [comment]
非常非常需要一个input插槽实现自定义值功能，非常感谢

**Comment by kooriookami** [comment]
可以自定义标签显示
https://element-plus.org/zh-CN/component/select.html#%E8%87%AA%E5%AE%9A%E4%B9%89%E6%A0%87%E7%AD%BE-1

**Reply by lhz239996423** [reply]
非常感谢您的回复，这帮助到了我，它非常有用！
